app/js/ContactForm.js

```diff
@@ -41,3 +40,12 @@ export class ContactForm {
     });
   }
 }
+
+function createMessageData($form) {
+  let serializedForm = $form.serializeArray();
+  let messageData = {};
+  serializedForm.map(field => {
+    messageData[field.name] = field.value;
+  });
+  return messageData;
+}
```

Comment:
I don't mind doing it that way, I just took the original piece of code. Could've boy scouted a bit more. It doesn't necessarily make it less readable, so I'll do it like that as a precedent ;-).